.github/workflows/conformance-datapath.yaml

```diff
@@ -428,7 +428,7 @@ jobs:
             ./cilium-cli install ${{ steps.vars.outputs.cilium_install_defaults }}
 
             ./cilium-cli status --wait
-            kubectl -n kube-system exec -it daemonset/cilium  -- cilium status
+            kubectl -n kube-system exec daemonset/cilium -- cilium status
```

Comment:
Interesting. I wonder why it exits with 0 though. Does it actually run the connectivity test now? :)

In your case in `test.sh` nothing is displayed, because you put your script to background, and as it wants the stdin, it waits in the background under `sigstop`. (Do a `fg` in bash and it will display you the `test` string and the script will terminate.)

Reply:
> In your case in `test.sh` nothing is displayed, because you put your script to background, and as it wants the stdin, it waits in the background under `sigstop`. (Do a `fg` in bash and it will display you the `test` string and the script will terminate.)

Yep. What I'm not entirely clear on is why when doing `fg` "after" doesn't show.

Reply:
It does for me
```
$ ./z.sh &
[1] 49590
$ before

[1]+  Stopped                 ./z.sh
$ fg
./z.sh
test
after
```

Anyway, if this fixes the issue, then fine.